content/collections/analytics/en/anomaly-forecast.md

```diff
@@ -63,7 +63,7 @@ You can also request an emailed digest of all anomalies in a project. These dige
 
 To set up automated alerts for anomalies in your projects, navigate to *Organization settings > Projects*, find the project you're interested in, and click it. Then click the *Automatic Alerts* tab and flip the toggle switch to subscribe to project alerts. 
 
-  ![auto-alerts.jpeg](/docs/output/img/analytics/auto-alerts.jpeg)
+  ![auto-alerts.jpeg](/docs/output/img/analytics/auto-alerts.png)
```

Comment:
image is broken
<img width="961" alt="Screenshot 2024-12-13 at 5 10 30 PM" src="https://github.com/user-attachments/assets/ace812ba-086b-4f63-93f9-c13dbeae5af6" />

https://github.com/amplitude/amplitude-docs/blob/main/public/docs/output/img/analytics/auto-alerts.png